.cirrus.yml

```diff
@@ -18,7 +18,7 @@ freebsd_12_task:
   install_script: pkg install -y bash git postgresql-libpqxx pkgconf libxml2 gmake perl5 libyaml p5-YAML-LibYAML rsync
 
   script:
-    - perl ${CIRRUS_WORKING_DIR}/test/test.pl --no-gen --make-cmd=gmake --vm=none --vm-max=2 --no-coverage --no-valgrind --module=command --test=backup
+    - cd .. && perl ${CIRRUS_WORKING_DIR}/test/test.pl --no-gen --make-cmd=gmake --vm=none --vm-max=2 --no-coverage --no-valgrind --module=command --test=backup
```

Comment:
`--vm=none` would seem to make `--vm-max` a no-op?

Reply:
--vm-max will still run things in parallel even when --vm=none. The option names have gotten a bit creaky -- they date back to when all this was done with vagrant VMs.